.github/workflows/ci.yml

```diff
@@ -263,17 +268,18 @@ jobs:
         shell: bash
 
       - name: Test and build frontend
-        uses: grafana/plugin-ci-workflows/actions/plugins/frontend@main
+        uses: grafana/plugin-ci-workflows/actions/plugins/frontend@giuseppe/gha-zizmor # zizmor: ignore[unpinned-uses]
 
       - name: Test and build backend
         if: ${{ steps.check-for-backend.outputs.has-backend == 'true' }}
-        uses: grafana/plugin-ci-workflows/actions/plugins/backend@main
-        with:
-          github-token: ${{ steps.generate-github-token.outputs.token }}
+        uses: grafana/plugin-ci-workflows/actions/plugins/backend@giuseppe/gha-zizmor # zizmor: ignore[unpinned-uses]
+        # TODO: Temporariy disabled until IT re-enables the GitHub App.
+        #with:
+        #  github-token: ${{ steps.generate-github-token.outputs.token }}
```

Comment:
what's the consequence of running this without token, no signature?

Reply:
This was once against just for testing, I have reverted it